syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -224,11 +232,20 @@ def zero(self):
         zero = (
             torch.zeros(*shape)
             .long()
-            .share(*self.locations, field=self.field, crypto_provider=self.crypto_provider)
-            .child
+            .share(
+                *self.locations, field=self.field, crypto_provider=self.crypto_provider, **no_wrap
+            )
         )
         return zero
 
+    def refresh(self):
```

Comment:
same as above

Reply:
I believe it is answered: it overwrites the default behaviour of refresh() for AST